Pipfile

```diff
@@ -54,6 +54,7 @@ python-dotenv = "*"
 importlib-resources = "*"
 dictdiffer = "*"
 setuptools = "*"
+pillow = "*"
```

Comment:
consider [this solution](https://stackoverflow.com/a/19035508) to avoid adding dependencies

Reply:
Trying this library:
https://pypi.org/project/imagesize/